src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -4132,6 +4137,9 @@
     <Compile Update="$(MSBuildThisFileDirectory)Windows_UI_Xaml\ThemeResources\BasicThemeResources.xaml.cs">
       <DependentUpon>BasicThemeResources.xaml</DependentUpon>
     </Compile>
+    <Compile Update="C:\GitRepos\_GitHub\Uno\src\SamplesApp\UITests.Shared\Resources\StaticResource\StaticResource_Simple.xaml.cs">
```

Comment:
There shouldn't be absolute paths (`C:\GitRepos` etc) in the project file - this is incorrectly added by recent versions of Visual Studio. Please remove it